controller/workitem_blackbox_test.go

```diff
@@ -1087,11 +1087,23 @@ func (s *WorkItem2Suite) TestWI2UpdateWorkItemType() {
 		compareWithGoldenAgnostic(t, filepath.Join(s.testDir, "update", "workitem_type.res.payload.golden.json"), newWI)
 	})
 
-	s.T().Run("disallow update of field along with type", func(t *testing.T) {
+	s.T().Run("allow update of field along with type", func(t *testing.T) {
 		u.Data.Attributes[workitem.SystemTitle] = "xyz"
-		// TODO (ibrahim) - Check type of error once error 422 has been added.
-		//https://github.com/fabric8-services/fabric8-wit/pull/2202#discussion_r210184092
-		test.UpdateWorkitemConflict(t, svc.Context, svc, s.workitemCtrl, fxt.WorkItems[0].ID, &u)
+		u.Data.Attributes[workitem.SystemVersion] = fxt.WorkItems[0].Version + 1
```

Comment:
I do not like the way type change tests are arranged. @kwk do you know a better way to do this? 
Also, this test cannot be executed independently because of `version = version + 1`.

Reply:
I've fixed this in https://github.com/fabric8-services/fabric8-wit/pull/2254/commits/45b0e0a6b19f9cc372549c31aef0bd0774763719